File: athrill-account-server/Libraries/Model/Users/UserTrackRecordModel.cs
using AT.Common.Enum;
using AT.Common.Helpers;
using System;

namespace AT.Model.Users
{
    public class UserTrackRecordModel
    {
        public int UserTrackRecordId { get; set; }
        public int UserId { get; set; }
        public int UserTrackTypeId { get; set; }
        public string IpAddress { get; set; }
        public string ClientName { get; set; }
        public int CreatedById { get; set; }
        public DateTime CreatedOn { get; set; }

        #region Link
        public UserModel User { get; set; }
        #endregion

        public UserTrackTypes UserTrackType;

        public string UserTrackTypeDisplayName 
        { 
            get 
            {
                return UserTrackType.ToString();
            } 
        }
    }
}

File: athrill-account-client/projects/at-models/src/lib/users/user-track-record.model.ts
import { UserModel } from './user.model';
import { UserTrackType } from '../../constants/at-enums.enum';

export class UserTrackRecordModel
{
    public UserTrackRecordId : number;
    public UserId : number;
    public UserTrackTypeId : number;
    public IpAddress : string;
    public ClientName : string;
    public UserTrackTypeDisplayName : string;
    public CreatedById :number;
    public CreatedOn : Date;

    public User: UserModel;
    public UserTrackType : UserTrackType;
}